src/library/trackset/setlogfeature.cpp

```diff
@@ -17,6 +17,8 @@
 #include "widget/wlibrarysidebar.h"
 #include "widget/wtracktableview.h"
 
+const int kNumDirectHistoryEntries = 5;
```

Comment:
This should be in a anonymous namespace, to hide it from the linker. 
It should be also a constexpr, which is really const without exception.